data/sitesEN.json

```diff
@@ -395,6 +395,17 @@
     "destination_platform": "mediawiki",
     "destination_icon": "tearsofthemis.png"
   },
+  {
+  "id": "en-touhou",
+  "origin": "Touhou Fandom Wiki",
+  "origin_base_url": "touhou.fandom.com",
+  "origin_content_path": "/wiki/",
+  "destination": "Touhou Wiki",
+  "destination_base_url": "https://en.touhouwiki.net/wiki/Touhou_Wiki",
```

Comment:
Thank you for the submission! Please update this to just `en.touhouwiki.net`

Reply:
Okay, just updated.